### PR TITLE
Updated Shaders page with zfast shaders and various fixes.

### PR DESCRIPTION
**Pull request by dankcushions**
For some reason the new images don't always show up when I preview the file - not sure why?

### CODE REVIEW

File: docs/Shaders-and-Smoothing.md [hunk 1]
```diff
@@ -42,42 +46,55 @@ Shaders are small programs that a dedicated graphics chip (GPU) runs to alter th
...
+You can select either preset via the [Configuration Editor](Configuration-Editor). Choose **Configure basic libretro emulator options**, then choose a specific emulator or **Configure default options for all libretro emulators** to make the change system-wide. Then, set `Video Shader Enable` to `true` and then set `Video Shader File` to the desired shader.
+
+You can can save per-game (or per-core) within a game via the **RGUI** with player 1's [Hotkey](Controller-Configuration#Hotkey) combination **Hotkey+X**, then **Quick Menu** > **Shaders**:
+
+![](https://i.imgur.com/EOCVSbT.png?1)
+
+Presets are beeing saved under the following folder:
 
-3. `crt-pi-vertical.glslp`
+`configs\all\retroarch\shaders\presets\`
```
Comment (cmitu) [inline docs/Shaders-and-Smoothing.md:76]: This has changed - the presets are saved in `configs\all\retroarch\config\<CORE_NAME>\<GAME_NAME>.glslp`.

File: docs/Shaders-and-Smoothing.md [hunk 2]
```diff
@@ -42,42 +46,55 @@ Shaders are small programs that a dedicated graphics chip (GPU) runs to alter th
...
+**Note:** These shaders can only be used in Libretro emulators. Most default emulators in RetroPie are Libretro emulators, but you can confirm by looking for the `lr-` prefix in the emulator name via the [Runcommand](Runcommand).
 
-Some arcade games were run on a vertical CRT; literally a regular TV flipped over on its side. Whilst libretro shaders  automatically rotate for this, the shadow mask effect does not, causing an unsightly 'rainbow' effect. Rather than add logic to the shader (making it slower), a special version for vertical games is available:
+You can select either preset via the [Configuration Editor](Configuration-Editor). Choose **Configure basic libretro emulator options**, then choose a specific emulator or **Configure default options for all libretro emulators** to make the change system-wide. Then, set `Video Shader Enable` to `true` and then set `Video Shader File` to the desired shader.
+
+You can can save per-game (or per-core) within a game via the **RGUI** with player 1's [Hotkey](Controller-Configuration#Hotkey) combination **Hotkey+X**, then **Quick Menu** > **Shaders**:
+
+![](https://i.imgur.com/EOCVSbT.png?1)
```
Comment (cmitu) [inline docs/Shaders-and-Smoothing.md:72]: The screenshot could be updated, there's a single 'Save' menu now, with multiple options on the next menu: https://imgur.com/a/gEK4IWF

File: docs/Shaders-and-Smoothing.md [hunk 3]
```diff
@@ -42,42 +46,55 @@ Shaders are small programs that a dedicated graphics chip (GPU) runs to alter th
 
 The second two presets are `crt-pi-vertical.glslp` and `crt-pi-curvature-vertical.glslp`, described in [Vertical games](#vertical-games)
 
-## Installation
+#### zfast
+
+[@ghogan42](https://retropie.org.uk/forum/user/ghogan42) has built shader **zfast**, which seems to perform even better. Five presets are included in RetroPie:
```
Comment (cmitu) [inline docs/Shaders-and-Smoothing.md:51]: maybe change to 
..._created/developed the **zfast** shader_..

**Comment by dankcushions** [comment]
Agreed with and applied changes :)

**Comment by cmitu** [comment]
Applied, thank you !

**Pull request merged**